package.json

```diff
@@ -16,7 +16,10 @@
     "test:console": "pf-codemods --v6 --no-cache test/console/frontend",
     "test:integreatly": "pf-codemods --v6 --no-cache test/tutorial-web-app",
     "test:packages": "yarn get:packages && node --unhandled-rejections=strict packages/pf-codemods/index.js --v6 --no-cache test/packages",
-    "test:classnames": "yarn build:classnames && lerna run test --scope=@patternfly/class-name-updater --stream",
+    "test:classnames": "yarn test:classnames:v5 && test:classnames:v6:nobuild",
+    "test:classnames:v5": "yarn build:classnames && lerna run test --scope=@patternfly/class-name-updater --stream",
+    "test:classnames:v6": "yarn build:classnames && lerna run test:v6 --scope=@patternfly/class-name-updater --stream",
+    "test:classnames:v6:nobuild": "lerna run test:v6 --scope=@patternfly/class-name-updater --stream",
```

Comment:
Then this can be
```suggestion
    "test:classnames": "lerna run test --scope=@patternfly/class-name-updater --stream",
    "test:classnames:v5": "yarn build:classnames && lerna run test:v5 --scope=@patternfly/class-name-updater --stream",
    "test:classnames:v6": "yarn build:classnames && lerna run test:v6 --scope=@patternfly/class-name-updater --stream",
```